### DISCUSSION TITLE
Ways to update the knowledge

### DISCUSSION THREAD

**Discussion by thelege2nd** [Pythagora-io/gpt-pilot#575] [Ideas]
Hello, Please add a step before project initiation to either allow us to give new updates to the GPT pilot, for example, latest API documentation files like PDFs or perhaps it can pull the information from a given link to use it. It should be part of the questionnaire steps because it creates projects with outdated information and wrong practices.

**Comment by Mlcastor** [comment]
yes please !!! i'm tired of correcting his obselete code. (it used davinci-text-003 wich is no longer supported since 2022)

**Comment by 80Builder80** [comment]
I agree. I know we are dealing with the training data for ChatGPT 4, but it's very frustrating to get deep into a project and run into code issues due to a lack of updated information. Being able to add updated documentation would be a great benefit. Users should be able to upload the most recent documents at any time during the process simply because the user may not anticipate a potential issue at the start of the project.

**Comment by blarvin** [comment]
#900
